deb-get

```diff
@@ -1260,6 +1261,10 @@ function deb_deb-get() {
     SUMMARY="'apt-get' functionality for .debs published in 3rd party repositories or via direct download package."
 }
 
+#### MAIN ####
+
+export WGET_DEFAULT_VERBOSITY="-q"
```

Comment:
maybe `export ${WGET_DEFAULT_VERBOSITY:="-q"} `
so that setting WGET_DEFAULT_VERBOSITY in the environment is passed in and not overridden (and same for timeout) ...

Reply:
makes absolute sense, I just thought a commandline option with getopt is the right way to do it.
NM, I'll add that, won't hurt.

Reply:
But I was wrong - export barfs on that and you need to do the defaulting separately (see below) Apologies for breaking your contribution.